GameRoomTest/game-room-frontend
# Improvement/2048 score and board

# Problem
We need to create the 2048 game sections.
- Scores section
- Board

# Solution
- Create a store (with zustand) for the game 2048 (useGame2048Store)
- Create the Scores component that gets the values from the game-2048 store.
- Create the Board component.
  - Create board with initial values (2 values in random positions).

# Test
1. Go to [2048 new game](https://staging.d23b8h4cpmeb82.amplifyapp.com/play/2048)
2. Check that the score and board components are displayed (no functionality so far)
3. Check that every time you enter new game, the board randomly renders 2 items with the value "2".
